allauth_2fa/forms.py

```diff
@@ -13,6 +14,22 @@
 }
 
 
+class _TokenToOTPTokenMixin:
+    exception = ImproperlyConfigured(
+        "The field `token` in TOTPDeviceRemoveForm has been renamed " "to `otp_token`.",
+    )
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+        if "token" in self.fields or "token" in self.data:
+            raise self.exception
+
+    @property
+    def token(self):
+        raise self.exception
```

Comment:
`exception` is too broad, and it had a hard-coded `TOTPDeviceRemoveForm` in it... How about e.g.

```suggestion
class _TokenToOTPTokenMixin:
    def __init__(self, **kwargs):
        super().__init__(**kwargs)

        if "token" in self.fields or "token" in self.data:
            self._raise_token_exception()

    @property
    def token(self):
        self._raise_token_exception()
        
    def _raise_token_exception(self):
        raise ImproperlyConfigured(f"The field `token` in {self} has been renamed to `otp_token`.")
```

Reply:
Good catch!